app/src/main/res/values/colors.xml

```diff
@@ -6,4 +6,5 @@
     <color name="titleTextColor">#bb0414</color>
     <color name="barBackgroundColor">#263238</color>
     <color name="yellow">#ffff00</color>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/d98de3fb9ea0ace0e5c0241379c958a7804774f9/app/src/main/res/values/colors.xml#L8"></span>The resource <code>R.color.yellow</code> appears to be unused</td>
    </tr>
  </tbody>
</table>